### PR TITLE
Feat/multiple vars returned

### LINKED ISSUE #49: [Bug] Unable to satisfy indentation rules for functions with multiple return values
I have a particular line in our contract that is unable to pass the linters indentation rules:

```
(bytes32 r, bytes32 s, uint8 v) = SomeLibrary.splitSignature(signature);
```
Error:

```
Expected indentation of 12 spaces, but found 13 [indent]
```

If you remove a space from before the first bracket, it replaces 13 with 11 in the error message.  Seems to be an issue with any function that has multiple return values.

### PR DESCRIPTION
**Pull request by canufeel**
Updated `solidity-antlr4`and generated new grammar to fix #49

### CODE REVIEW

**Comment by canufeel** [comment]
@fvictorio I can spend some times to add couple of tests to this if you want to make sure regression wont be possible in the future. Though not sure to where they belong in terms of test modules. Have not found any variable declaration tests in solhint

**Comment by fvictorio** [comment]
Hi @canufeel and thanks a lot for contributing!

Maybe we could add some tests to check that the `no-unused-vars` rule works with this form of variable declaration? These tests are in the `test/best-practise-rules.js` file. What do you think?

**Comment by canufeel** [comment]
@fvictorio friendly ping. I have added some number of tests for this PR. Let me know if you feel something else is needed.

File: test/align-rules.js [hunk 1]
```diff
@@ -80,6 +80,50 @@ describe('Linter', function () {
...
+                '    }                                              ',
+                '}                                                  '
+            );
+
+            const report = linter.processStr(code);
+
+            assert.equal(report.errorCount, 0);
+        });
+
+        it('should not raise error for multiline multi variable functions with no additional indent', function () {
```
Comment (fvictorio) [inline test/align-rules.js]: I'm not sure about this. Is that indentation correct? Or is just that the indentation rule does not understand the tuple assignment and allows anything?

In any case, it's not related to this PR, but I'd like to know if this test should be removed or not.

**Comment by fvictorio** [comment]
I'm going to merge this and create a issue for the alignment of multiple variables.

**Comment by fvictorio** [comment]
Thanks a lot @canufeel, and sorry for taking so long to merge this.

**Pull request merged**